src/components/layout.js

```diff
@@ -29,7 +29,8 @@ const Layout = ({ children }) => {
       <div
         style={{
           margin: `0 auto`,
-          maxWidth: `var(--size-content)`,
+          // JDY: Removed below line so can extend to edges of browser window
+          //maxWidth: `var(--size-content)`,
```

Comment:
Delete any commented out files.

Reply:
Another line I would remove in production, but want to keep for my own knowledge in this project, as this was an original line from the generated code.